downstream.go

```diff
@@ -57,6 +57,7 @@ var permanentDownstreamCaps = map[string]string{
 	"cap-notify":   "",
 	"echo-message": "",
 	"message-tags": "",
+	"multi-prefix": "",
```

Comment:
Unfortunately we can't advertise multi-prefix unconditionally. We can only advertise it if all upstream connections support it.

See how away-notify is handled.